index.bs

```diff
@@ -2632,10 +2632,11 @@ run the following steps:</p>
 
       <ol>
 
-       <li><p>If |value| contains a U+002C COMMA character (,), then let |linepos| be the leading
-       substring of |value| up to and excluding the first U+002C COMMA character (,) in that string
-       and let |linealign| be the trailing substring of |value| starting from the character
-       immediately after the first U+002C COMMA character (,) in that string.</p></li>
+       <li><p>If |value| contains a U+002C COMMA character (,) that is not the last character in
```

Comment:
No. ^_^ This is ugly as it is, let's not consider making it worse. But even so, XYZ is invalid now and should still cause the whole thing to be dropped.